ui/perfherder/compare/CompareTableView.jsx

```diff
@@ -272,6 +272,7 @@ export default class CompareTableView extends React.Component {
                             : 'Perfherder Compare Revisions'
                         }
                         pageTitleQueryParam={pageTitle}
+                        defaultPageTitle={window.document.title}
```

Comment:
We have `Helmet` library installed for updating the document title rather than doing it this way.

Reply:
Oh! Thats great!
Should I use replace [this](https://github.com/mozilla/treeherder/blob/13a6788c0dd8e1605524c9c90835242bccab7c70/ui/perfherder/compare/CompareTableView.jsx#L142) by the helmet package?

Reply:
Ah! I had a doubt @ionutgoldan  in what you said 
Even if we encapsulate the original title in a state vaiable in CompareTableView,
We need to paas it as props to ComparePageTitle and use it there because the change in title is happening in ComparePageTitle Component actually

Reply:
> 
> 
> Ah! I had a doubt @ionutgoldan in what you said
> Even if we encapsulate the original title in a state vaiable in CompareTableView,
> We need to paas it as props to ComparePageTitle and use it there because the change in title is happening in ComparePageTitle Component actually

What I meant is avoid assigning the computed string to `window.document.title` directly; we want to avoid this practice from now on. Simply store the string in a variable, which you then pass to ComparePageTitle component, via its new `defaultPageTitle` param.

The `Helmet` library will take care of updating the `window.document.title` under the hood.